### ISSUE TITLE
Start both projects?

### ISSUE THREAD

**Issue by Rod-in-NM** [Azure-Samples/active-directory-dotnet-native-desktop#26]
After setting up the ToDoListClient and ToDoListService, the instructions told the reader to set both projects as startup projects (which I did) and then run it. How do I run it, without VS 2017 wanting to debug it?

**Comment by Rod-in-NM** [comment]
I'm sorry, the instruction you linked to do not work. I've followed them, but I keep getting this error: 
![2018-06-04](https://user-images.githubusercontent.com/13875530/40927151-3d3dbf20-67db-11e8-8635-b9d7770bb2fe.png)

**Comment by jmprieur** [comment]
Yes  … a VS bug ... apprently, this was no longer working with VS 2017 15.6.
I've installed VS 2017 15.7.2, and it's working fine for me now.
